osi_hostvehicledata.proto

```diff
@@ -2,31 +2,281 @@ syntax = "proto2";
+    //
+    message VehicleWheels
+    {
+        // Description of each wheel.
+        //
+        // \note OSI uses singular instead of plural for repeated field names.
+        //
+        repeated WheelData wheel_data = 1;
+
+        //
+        // \brief The focus here is on the description of a wheel.
+        //
+        message WheelData
+        {
+            // The axle which contains this wheel. A value of 0 represents the
+            // front-most axle of the vehicle with higher numbers incrementing
+            // towards the rear-most axle.
+            //
+            optional uint32 axle = 1;
+
+            // The index of the wheel on the axle, counting in the direction
+            // of positive-y, that is, right-to-left.
+            //
+            // For example, on a standard 2-axle, 4-wheel car, the rear-right
+            // wheel would be (axle=1, index=0).
+            // This concept works also for twin tires.
+            //
+            optional uint32 index = 2;
+
+            // Rotation rate of the wheel based on the processed output of the hall sensor measurements at the wheel.
+            // The rotation rate around the y-axis with respect to the wheel's coordinate system.
+            // 
+            // Unit: rad/s.
+            //
+            // The sign convention is defined using the right-hand rule.
+            // It is applied on the y-axis of the vehicle's reference system (center of bounding box).
+            // Counterclockwise is positive and clockwise is negative.
+            //
+            // \image html OSI_RotationRate.svg
+            // \note The vehicle's reference coordinate system is only used to determine the sign convention of the rotation rate.
+            //
+            optional double rotation_rate = 3;
+
+            // Contains the longitudinal, measured slip of the tire.
+            // \par References:
+            // - https://www.kfz-tech.de/Biblio/Formelsammlung/Schlupf.htm
+            //
+            // Unit: %
+            //
+            // The sign convention is defined using the right-hand rule.
+            // It is applied on the y-axis of the vehicle's reference system (center of bounding box).
+            // Counterclockwise is positive and clockwise is negative.
+            //
+            optional double slip = 4;
+        }
+    }
+
+    //
+    // \brief Current calculated and estimated location that can be based on GNSS- and related navigation sensors,
+    // but this message does not contain the single sensor values of the sensorics.
+    //
+    // This message contains the most accurate information the vehicle knows about its positioning and kinematics
+    // available on the board net.
+    // Because of this the values can differ from the "true" values calculated out of
+    // GroundTruth::proj_string, GroundTruth::MovingObject::BaseMoving::position, GroundTruth::host_vehicle_id.
+    //
+    message VehiclePositionAndKinematics
+    {
```

Comment:
maybe mention in the description that toolboxes exist that can convert ecef2enu, enu2ecef, ecef2geodetic, geodetic2ecef,...